File: Modul 6/NO 4/PRAK504-2210817310016-Riyo Aurora Guison.c
#include<stdio.h>
int reverse(int angka){
    int a=0;
    while(angka != 0){
        a = a*10;
        a = a+angka%10;
        angka = angka/10;
    }
    return a;
}
int main(){
    int A,B;
    scanf("%d %d",&A,&B);
    A=reverse(A);
    B=reverse(B);
    int C=A+B;
    printf("%d",reverse(C));
}

File: Modul 6/NO 4/PRAK504-2210817310016-Riyo Aurora Guison.py
def reverse(angka):
    i = 0
    while angka != 0:
        i = (i*10) + angka % 10
        angka //= 10
    return i
for i in range (3):
    A,B = input().split()
    A = reverse(int(A))
    B = reverse(int(B))
    C = A+B
    print(f'{reverse(C)}\n')